camunda-formio-tasklist-vue/src/components/addons/ExpandContract.vue

```diff
@@ -1,37 +1,59 @@
 <template>
-  <b-navbar>
-    <b-nav-form>
-      <b-button @click="clickMaxi" variant="light" size="sm">
-        <i
-          v-if="maximize"
-          class="fa fa-expand"
-          v-b-tooltip.hover
-          title="Click to expand the form"
-        ></i>
-        <i
-          v-else
-          class="fa fa-compress"
-          v-b-tooltip.hover
-          title="Click to see the task list"
-        ></i>
-      </b-button>
-    </b-nav-form>
-  </b-navbar>
+  <button
```

Comment:
As @shuhaib-aot reported, when I tried by installing in top of forms-flow-ai-extensions, I am not able to see any tasks at all.

![image](https://user-images.githubusercontent.com/70306694/156015098-350a0dfd-b750-4c07-b103-1410d1b06be3.png)

It was actually visible [till this commit](https://github.com/AOT-Technologies/forms-flow-ai-extensions/pull/158/commits/d05ffb045f4140901ed8501da6f353808c437f67). It may have been because our parent application didn't have font-family setup.

Reply:
In our applications but font-family is not present in parent application. That's why our CSS maybe broken.

Reply:
something seems off the way you guys are trying, the screenshot the description is how the app will be looking from the parent application. lets setup a call and I will help you to run it correctly